src/main/java/nextstep/qna/domain/Answers.java

```diff
@@ -0,0 +1,33 @@
+package nextstep.qna.domain;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import nextstep.qna.CannotDeleteException;
+import nextstep.users.domain.NsUser;
+
+public class Answers {
+
+    private final List<Answer> answers;
+
+    public Answers(List<Answer> answers) {
+        this.answers = answers;
+    }
+
+    public void postHasAnswersWrittenByMe (NsUser nsUser) throws CannotDeleteException {
+        for(Answer answer : answers) {
```

Comment:
코드 포맷팅 확인해주시면 좋을 것 같습니다.
```suggestion
    public void postHasAnswersWrittenByMe(NsUser nsUser) throws CannotDeleteException {
        for (Answer answer : answers) {
```